### ISSUE TITLE
Create Contact page

### ISSUE THREAD

**Issue by sscovil** [sscovil/something-awesome#10]
In order for the `Contact` link to work in our side navigation menu, we need to create the page. To do this, we will copy the contents of [index.html](https://github.com/sscovil/something-awesome/blob/master/src/public/index.html) (after #7 is complete) into a new file in the same directory called `contact.html`. In the new file, we will replace the contents of the `<main>` section with a web form. Everything else will remain the same. The CSS rules we created in #8 should also apply to this page, since we target HTML elements with those CSS rules and both HTML files have the same structure.

The contact form will contain the following elements:

- [x] name text input
- [x] email text input
- [x] subject line text input
- [x] message textarea
- [x] submit button

For now, we'll just focus on getting the HTML elements correct. We will wire up the form to our REST API in a separate story. Likewise, we'll add form validation (i.e. errors that indicate a required field must be filled in before the form can be submitted) and feedback (i.e. clearing the form and displaying a message upon submission) in a separate story.

There are plenty of tutorials out there on building web forms, if you need one. This looks decent: https://code.tutsplus.com/tutorials/build-a-neat-html5-powered-contact-form--net-20426

**Comment by sscovil** [comment]
This is another good resource for understanding how web forms can be used to pass data to the server: https://developer.mozilla.org/en-US/docs/Learn/HTML/Forms/Sending_and_retrieving_form_data

**Comment by sscovil** [comment]
Also, be sure the `<form>` element includes these properties:

`action="/api/forms/contact" method="POST"`
